docs/DeveloperGuide.md

```diff
@@ -551,7 +483,31 @@ testers are expected to do more *exploratory* testing.
     2. Test case: `delete 1`<br>
       Expected: No client is deleted. Error detail informing of environment discrepancy shown in the status message.
 
+### Finding clients
+
+1. Finding a client in the client list view
+
+   1. Prerequisites: List all clients using the `list` command. Multiple clients in the list.
+
+   2. Test case: `find Alex`<br>
+      Expected: All clients whose name or company contain Alex are shown. Details of the search shown in the status message.
+
+   3. Test case: `find`<br>
+      Expected: Error details informing of invalid command format shown in the status message.
+
+   4. Test case: `find fhodolkg`<br>
```

Comment:
I'm being very nitpicky but maybe you can actually say out the names of the clients in the list. It wasn't specified there wasn't a client named `fhodolkg`

Reply:
Hmmm maybe I remove this test case?